localization/en.ftl

```diff
@@ -185,3 +185,8 @@ pillbox-menus-desc = Show the buttons on the right side, such as the Video Creat
 ## Manage Metadata
 manage-metadata-name = Manage Metadata (Core)
 manage-metadata-desc = Manage Metadata, such as GLesmos or pinned/unpinned status
+
+## Intellisense
+intellisense-name = Intellisense
+intellisense-desc = Brings several common IDE features to Desmos, including autocompletion suggestions, function call help, and jump to definition. Documentation here:
+intellisense-jump2def-menu-instructions = has multiple definitions. Pick one to jump to below. Press ESC to close this menu.
```

Comment:
Don't need the "Press ESC to close this menu." It's a pretty common interaction

Reply:
Fixed